### ISSUE TITLE
TypeError: attrib() got an unexpected keyword argument 'convert' on import

### ISSUE THREAD

**Issue by robobeaver6** [weaveworks/grafanalib#186]
## What you expected to happen?
import grafanalib fails with attrs 19.3
downgrading attrs to 19.1 and it imports fine.
## What happened?

```
>>>from grafanalib.core import *
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/usr/share/java/pycharm/helpers/pydev/_pydev_bundle/pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "/home/rhall/.virtualenvs/nornir-netops-6ImgWdqL/lib/python3.6/site-packages/grafanalib/core.py", line 449, in <module>
    class Row(object):
  File "/home/rhall/.virtualenvs/nornir-netops-6ImgWdqL/lib/python3.6/site-packages/grafanalib/core.py", line 452, in Row
    panels = attr.ib(default=attr.Factory(list), convert=_balance_panels)
TypeError: attrib() got an unexpected keyword argument 'convert'
```
## How to reproduce it?

Current versions:
```
Package               Version   Location                                                                               
--------------------- --------- ---------------------------------------------------------------------------------------
appdirs               1.4.3     
argon2-cffi           19.1.0    
arrow                 0.15.2    
asn1crypto            1.0.1     
atomicwrites          1.3.0     
attrs                 19.3.0    
bcrypt                3.1.7     
black                 19.3b0    /data/home/rhall/.local/share/virtualenvs/nornir-netops-6ImgWdqL/src/black             
certifi               2019.9.11 
cffi                  1.13.1    
chardet               3.0.4     
ciscoconfparse        1.4.7     
Click                 7.0       
colorama              0.4.1     
construct             2.9.45    
coverage              4.5.3     
cryptography          2.8       
dataclasses           0.7       
dnspython             1.16.0    
future                0.18.1    
grafanalib            0.5.3     
graphviz              0.13      
hdb-client            0.3       /data/home/rhall/.local/share/virtualenvs/nornir-netops-6ImgWdqL/src/hdb-client/src/lib
hier-config           1.6.0     
hvac                  0.9.5     
idna                  2.8       
importlib-metadata    0.23      
ipaddress             1.0.23    
Jinja2                2.10.3    
junos-eznc            2.2.1     
lxml                  4.4.1     
MarkupSafe            1.1.1     
mock                  2.0.0     
more-itertools        7.2.0     
mypy-extensions       0.4.3     
napalm                2.5.0     
napalm-asa            0.1.1     
natsort               6.0.0     
ncclient              0.6.6     
netaddr               0.7.19    
netmiko               2.4.2     
nornir                2.3.0     
nxapi-plumbing        0.5.2     
packaging             19.2      
paramiko              2.6.0     
passlib               1.7.1     
pbr                   5.1.3     
pep8                  1.7.1     
pip                   19.0.3    
pluggy                0.13.0    
py                    1.8.0     
pyasn1                0.4.5     
pycparser             2.19      
pycryptodome          3.9.0     
pydantic              0.18.2    
pyeapi                0.8.2     
pygraphviz            1.5       
pyIOSXR               0.53      
pykeepass             3.0.3     
pykerberos            1.2.1     
PyNaCl                1.3.0     
pyparsing             2.4.2     
pyserial              3.4       
pytest                5.2.1     
pytest-mock           1.10.2    
python-dateutil       2.8.0     
PyYAML                5.1.2     
requests              2.22.0    
requests-kerberos     0.12.0    
ruamel.yaml           0.15.100  
scp                   0.13.2    
selectors2            2.0.1     
setuptools            40.8.0    
six                   1.12.0    
slumber               0.7.1     
textfsm               1.1.0     
toml                  0.10.0    
transitions           0.7.1     
typing-extensions     3.7.4     
urllib3               1.25.6    
wcwidth               0.1.7     
wheel                 0.33.1    
yamlordereddictloader 0.4.0     
zipp                  0.6.0 
```

**Comment by rendyokaIRL** [comment]
Looks like `convert` has been deprecated since version 19.2.0 according to the release notes:

https://www.attrs.org/en/stable/changelog.html#backward-incompatible-changes

**Comment by yunw** [comment]
to downgrade to 19.1.0:
` pip freeze | grep attrs  ## if the output is 19.3.0, then continues.
  pip install 'attrs==19.1.0' --force-reinstall
  pip freeze | grep attrs`

**Comment by bboreham** [comment]
Fixed by #180
